.gitignore

```diff
@@ -41,3 +41,10 @@ Thumbs.db
 
 # Next.js
 .next
+
+# dev database
+db/gddy.sqlite
+
+# env files
+.env
+.env.local
```

Comment:
I don't like having .env files in source control, so I removed them and added ignore rules.